mmv1/products/privateca/api.yaml

```
@@ -108,107 +117,248 @@ objects:
+                    - !ruby/object:Api::Type::NestedObject
+                      name: 'objectId'
+                      description: |
+                        Describes values that are relevant in a CA certificate.
+                      properties:
+                        - !ruby/object:Api::Type::Array
+                          name: 'objectIdPath'
+                          required: true
+                          item_type: Api::Type::Integer
+                          description: |
+                            An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
+              - !ruby/object:Api::Type::Array
+                name: 'policyIds'
+                description: |
+                  Describes the X.509 certificate policy object identifiers, per https://tools.ietf.org/html/rfc5280#section-4.2.1.4.
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::Array
+                      name: 'objectIdPath'
+                      required: true
+                      item_type: Api::Type::Integer
+                      description: |
+                        An ObjectId specifies an object identifier (OID). These provide context and describe types in ASN.1 messages.
+              - !ruby/object:Api::Type::Array
+                name: 'aiaOcspServers'
+                item_type: Api::Type::String
+                description: |
+                  Describes Online Certificate Status Protocol (OCSP) endpoint addresses that appear in the
+                  "Authority Information Access" extension in the certificate.
+              - !ruby/object:Api::Type::NestedObject
+                name: 'caOptions'
+                description: |
+                  Describes values that are relevant in a CA certificate.
+                properties:
+                  - !ruby/object:Api::Type::Boolean
+                    name: 'isCa'
+                    description: |
+                      Refers to the "CA" X.509 extension, which is a boolean value. When this value is missing,
+                      the extension will be omitted from the CA certificate.
+                  - !ruby/object:Api::Type::Integer
+                    name: 'maxIssuerPathLength'
+                    description: |
+                      Refers to the path length restriction X.509 extension. For a CA certificate, this value describes the depth of
+                      subordinate CA certificates that are allowed. If this value is less than 0, the request will fail. If this
+                      value is missing, the max path length will be omitted from the CA certificate.
+              - !ruby/object:Api::Type::NestedObject
+                name: 'keyUsage'
+                description: |
+                  Optional. Indicates the intended use for keys that correspond to a certificate.
```

```suggestion
                  Indicates the intended use for keys that correspond to a certificate.
```

nit: Implied by the field being optional